src/pages/createCohort/components/createCohortByName.jsx

```diff
@@ -5,26 +5,32 @@ import './style.css';
 
 const CreateCohortByName = () => {
   const [createCohortRes, setCreateCohortRes] = useState(false);
-  const [cohortName, setCohortName] = useState('');
+  const [name, setCohortName] = useState('');
 
   function createCohortName(event) {
     event.preventDefault();
     setCohortName(event.target.value);
+    
   }
+  console.log(name)
 
   function createCohort(event) {
     event.preventDefault();
     client
+
       .post('/cohort', { name: cohortName })
+
       .then(res => {
         if (res.data.status === 'success') {
           setCreateCohortRes(true);
+          
         }
       })
       .catch(console.log);
     setTimeout(() => {
       setCreateCohortRes(false);
     }, 3000);
+    window.location.reload()
```

Comment:
What's the reason for this reload?

Reply:
to refresh the page so the new name would show